src/node_url.cc

```diff
@@ -101,8 +101,7 @@ void BindingData::DomainToUnicode(const FunctionCallbackInfo<Value>& args) {
 
   std::string input = Utf8Value(env->isolate(), args[0]).ToString();
   if (input.empty()) {
-    return args.GetReturnValue().Set(
-        String::NewFromUtf8(env->isolate(), "").ToLocalChecked());
+    return args.GetReturnValue().Set(FIXED_ONE_BYTE_STRING(env->isolate(), ""));
```

Comment:
```suggestion
    return args.GetReturnValue().Set(String::Empty(env->isolate()));
```

Reply:
@bnoordhuis Thanks for the comment.
The current file also has code that uses the same.
It also has been fixed.
I would appreciate it if you could take a look if you have time.